cmd/helm/search/search.go

```diff
@@ -231,5 +232,5 @@ func (s scoreSorter) Less(a, b int) bool {
 func indstr(name string, ref *repo.ChartVersion) string {
 	i := ref.Name + sep + name + "/" + ref.Name + sep +
 		ref.Description + sep + strings.Join(ref.Keywords, " ")
-	return strings.ToLower(i)
```

Comment:
This change should be covered with unit tests (I think TestAddRepo_Sort should be altered for this).